.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+      "chef-software.chef",
+      "rebornix.ruby",
```

Comment:
I just got an error from VSCOde that it didn't exist! :(

Reply:
ruh roh. also newline @Xorima 😁

Reply:
This one is all @tas50 I just put it in the right location